build/dt-report-generator-bundle.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const browserify = require('browserify');
+const fs = require('fs');
+
+const distDir = __dirname + '/../dist';
+const outFile = `${distDir}/report-generator.js`;
+const generatorFilename = `./lighthouse-core/report/report-generator.js`;
+browserify(generatorFilename, {standalone: 'ReportGenerator'})
+  .transform('brfs')
+  .bundle((err, src) => {
+    if (err) throw err;
+    fs.writeFileSync(outFile, src.toString());
+  });
```

Comment:
sweet, I think this might help @exterkamp as well. At least I think he was looking for a standalone report generator?

Reply:
Cool! He can just remove the `dt-` bit then, that'd be a great PR @exterkamp  :)